Header header                                      # Contains timestamp information for the most recent update to the object list
bool cleared                                       # A flag indicating this message is the result of a clear instead of a segment
rail_semantic_grasping/SemanticObject[] objects    # The list of segmented objects
